deploy/main.tf

```diff
@@ -1,18 +1,21 @@
-// Resource group configuration
+data "local_file" "script" {
+  filename = "../src/List-PastAlerts.ps1"
```

Comment:
I would have named this `list_past_alerts_script`

Reply:
Updated